contracts/linear/src/events.rs

```diff
@@ -51,6 +51,37 @@ pub enum Event<'a> {
         new_balance: &'a U128,
         rewards: &'a U128,
     },
+    // Drain Operations
+    DrainUnstakeAttempt {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    DrainUnstakeSuccess {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    DrainUnstakeFailed {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    DrainWithdrawAttempt {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    DrainWithdrawSuccess {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    DrainWithdrawFailed {
+        validator_id: &'a AccountId,
+        amount: &'a U128,
+    },
+    // Sync validator balance
+    AccountBalanceSynced {
```

Comment:
Should we add Epoch prefix for `AccountBalanceSynced` here? Is it an epoch action?

Reply:
What about renaming to `BalanceFromValidatorSynced` to keep consistent with the function name `sync_balance_from_validator ` ?